ethcore/src/engines/authority_round/mod.rs

```diff
@@ -91,6 +98,8 @@ impl From<ethjson::spec::AuthorityRoundParams> for AuthorityRoundParams {
 			block_reward: p.block_reward.map_or_else(Default::default, Into::into),
 			maximum_uncle_count_transition: p.maximum_uncle_count_transition.map_or(0, Into::into),
 			maximum_uncle_count: p.maximum_uncle_count.map_or(0, Into::into),
+			empty_steps_transition: p.empty_steps_transition.map_or(u64::max_value(), |n| ::std::cmp::max(n.into(), 1)),
```

Comment:
why is it lower bounded at 1?

Reply:
The genesis block seal doesn't include the empty steps and this way we don't have to deal with it.